lib/api/CT-list-update-service.js

```diff
@@ -9,7 +9,7 @@ var CTListUpdateService = function(options) {
         'Cache-Control': 'no-cache'
     }
     this._apiToken = config.get().crowdtangle.apiToken;
-    this._outputDir = options.directory;
+    this._outputDir = options ? options.directory : '../../config/crowdtangle_list.json';
```

Comment:
Simplification: Instead of dynamically assigning this hardcoded string here, you could use below `CTListUpdateService.prototype.OUTPUT_DIR = '../../config/crowdtangle_list.json'` to make it static.

Then in the test file, you could import this class and change the static variable in the `stubWithFixture` method that already exists: something like `CTListUpdateService.prototype.OUTPUT_DIR = outputFixtureFile`.

This will allow us to use nearly the same logic in test and production, just with a different variable name (instead of using an entirely different test route which could lead to unintended bugs later on if we forget to keep them in sync).